plaso/parsers/wincc.py

```diff
@@ -0,0 +1,341 @@
+# -*- coding: utf-8 -*-
+"""Text parser plugin for WinCC log files."""
+
+from dfvfs.helpers import text_file
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.lib import errors
+from plaso.parsers import interface
+from plaso.parsers import manager
+
+
+class SIMATICS7EventData(events.EventData):
+  """SIMATIC S7 event data."""
+
+  DATA_TYPE = 'wincc:simatic_s7:entry'
+
+  def __init__(self):
+    """Initializes event data."""
+    super(SIMATICS7EventData, self).__init__(data_type=self.DATA_TYPE)
+
+    self.creation_time = None
+    self.message = None
+
+
+class WinCCSysLogEventData(events.EventData):
```

Comment:
Win => Windows ? Or is the name of the product WinCC ?

Reply:
It is called [WinCC](https://en.wikipedia.org/wiki/WinCC)